packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -198,7 +198,6 @@ const SuiteLayout = (props: SuiteLayoutProps) => {
                     </BodyNarrow>
                 )}
             </LayoutContext.Provider>
-            <BetaBadge />
```

Comment:
I'd recommend to remove also the component's source as well so it's not abandoned in the codebase, it's probably not needed anymore.

Reply:
removed with source, thx